docs/build/linux-ubuntu.md

```diff
@@ -53,21 +54,42 @@ sudo mk-build-deps --install linux/debian/control
 
 Node.js v18 is required for Core build, Web tests, and Developer command line tools.
 
+You can install it with:
+
+```shell
+curl -sL https://deb.nodesource.com/setup_18.x | bash
+apt-get -q -y install nodejs
+```
+
 ## Keyman for Linux
 
-All dependencies are already installed if you followed the instructions under [Prerequisites](#Prerequisites).
+All dependencies are already installed if you followed the instructions
+under [Prerequisites](#prerequisites).
 
-Building:
+### Building Keyman for Linux
 
-* [Building Keyman for Linux](../../linux/README.md)
+- [Building Keyman for Linux](../../linux/README.md)
 
 ## Keyman Core
 
-All dependencies are already installed if you followed the instructions under [Prerequisites](#Prerequisites).
+Most dependencies are already installed if you followed the instructions under
+[Prerequisites](#prerequisites). You'll still have to install `emscripten`:
 
-Building:
+```shell
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install latest
+./emsdk activate latest
+source ./emsdk_env.sh
```

Comment:
We don't want to put EMSDK onto the path, because it pollutes with its own version of node and python and other tools -- instead we want to set the EMSCRIPTEN_BASE environment variable to the root of EMSDK